plugins/filter_kubernetes/kube_meta.c

```diff
@@ -962,6 +989,10 @@ int flb_kube_meta_init(struct flb_kube *ctx, struct flb_config *config)
 
         /* Gather info from API server */
         flb_plg_info(ctx->ins, "testing connectivity with API server...");
+
+#ifdef FLB_SYSTEM_WINDOWS
```

Comment:
I think this is fine to keep it enabled for all OS, not just windows.

Reply:
OK. I made the feature available for all platforms.